File: DayTwo/Java/LevelOne/TaskTen.java
import java.util.Scanner;

public class TaskTen {

    public static void main(String[] args) {

        Scanner collectInput = new Scanner(System.in);

        System.out.println("Enter first number: ");
        int numberOne = collectInput.nextInt();

        System.out.println("Enter second number: ");
        int numberTwo = collectInput.nextInt();

        System.out.println("Enter third number: ");
        int numberThree = collectInput.nextInt();


        int average = (numberOne + numberTwo + numberThree) / 3;

        System.out.println("The average is: " + average);
    }

}

File: DayTwo/JavaScript/LevelOne/TaskTen.js
const prompt = require("prompt-sync")()

let numberOne = Number(prompt("Enter first number: "))

let numberTwo = Number(prompt("Enter second number: "))

let numberThree = Number(prompt("Enter third number: "))

let average = (numberOne + numberTwo + numberThree) / 3;

console.log("The average is: " + average);
